docs/contents/deployment.md

```diff
@@ -0,0 +1,65 @@
+# Deploying Books.jl to a Web Service
```

Comment:
```suggestion
# Deploying Books.jl to a web service
```
I haven't really used title case in the rest of the docs. I don't really mind, but since everything is already in not title case, It's probably easiest to stick to that.